src/main/java/com/example/Trello/repositories/CardRepository.java

```diff
@@ -9,5 +9,5 @@
 
 @Repository
 public interface CardRepository extends JpaRepository<CardEntity, Long> {
-    List<CardEntity> findCardEntitiesByBoardEntity(BoardEntity boardEntity);
+    List<CardEntity> findCardEntitiesByBoardEntityOrderByCreatedAtAsc(BoardEntity boardEntity);
```

Comment:
The newest card should be the first, the oldest one is the last in a resulting list.
I assume your result is from the oldest to newest. You need to change it to `Desc`. Correct me if I'm wrong